addon/adapters/firebase.js

```diff
@@ -502,8 +502,10 @@ export default DS.Adapter.extend(Ember.Evented, {
   _getRef: function(type, id) {
     var ref = this._ref;
     var typeName = type;
-    if (type && type.typeKey) {
-      typeName = type.typeKey;
+
+    // Fall to typeKey for Ember Data < 1.0beta.18
+    if (type && (type.modelName || type.typeKey)) {
```

Comment:
if I am not mistaken, `modelName` is now **dasherized**. Do we need a test to make sure that it still resolved `some-model` to `someModels` path so that existing data is not lost?

I think all our tests use single syllable names `post`, `comment`, etc